### DISCUSSION TITLE
Welcome to SyllabusX Discussions!

### DISCUSSION THREAD

**Discussion by akshat-OwO** [akshat-OwO/syllabusx-client#58] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jatingodnani** [comment]
Hello @akshat-OwO,i want to contribute to this project, would you tell on which part your are working.

**Reply by akshat-OwO** [reply]
Currently, We have not planned for anything new. However there are some things that you can look upon:
- Creating new themes
- Improving the subjectList component UI/UX (looks crammed)

**Comment by arpitrawat234** [comment]
Hey its down today ?

**Reply by akshat-OwO** [reply]
sorry was a dns issue should be solve now

**Reply by arpitrawat234** [reply]
Yeah its working now thanks
